base/dcp_client_test.go

```diff
@@ -385,3 +385,26 @@ func TestResumeStoppedFeed(t *testing.T) {
 	}
 
 }
+
+// TestBadAgentPriority makes sure we can not specify agent priority as high
+func TestBadAgentPriority(t *testing.T) {
+	if UnitTestUrlIsWalrus() {
+		t.Skip("This test only works against Couchbase Server, since DCPClient requires collections")
```

Comment:
There's nothing collection specific here (`AsCollection` means we're using gocbv2 against Couchbase Server, not necessarily that we're using a named collection)

```suggestion
		t.Skip("This test only works against Couchbase Server, since DCPClient requires a Couchbase DCP feed")
```

Reply:
DCPClient requires type `base.Collection`, which is not a named collection. Is there a better way to phrase this? i.e. it can't work with walrus because it can't be a cbdatasource.

Reply:
As discussed maybe just `base.Collection` for less ambiguity